Add downloads samples (#934)
GoogleChrome/chrome-extensions-samples: api-samples/downloads/download_manager/README.md

# chrome.downloads - Download Manager

> A sample that demonstrates how to use the [`chrome.downloads`](https://developer.chrome.com/docs/extensions/reference/downloads) API.

Reviewer: ```suggestion
A sample that demonstrates using the [`chrome.downloads`](https://developer.chrome.com/docs/extensions/reference/downloads) API.
```

## Overview

> In this sample, multiple [`chrome.downloads`](https://developer.chrome.com/docs/extensions/reference/downloads) APIs are comprehensively used to implement a simple download manager.

Reviewer: ```suggestion
This sample uses multiple [`chrome.downloads`](https://developer.chrome.com/docs/extensions/reference/downloads) APIs to implement a simple download manager.
```

Reviewer: 'Comprehensive' = covering completely. Does this sample really use every aspect of `chrome.downloads`?
